modules/publication/jsx/viewProject.js

```diff
@@ -257,10 +247,11 @@ class ViewProject extends React.Component {
           label="Lead Investigator"
           text={this.state.formData.leadInvestigator}
         />
-        <StaticElement
-         name="leadInvestigatorEmail"
-         label="Lead Investigator Email"
-         text={this.state.formData.leadInvestigatorEmail}
+        <EmailElement
```

Comment:
it looks to me that this EmailElement is the one from jsx/Form.js which is an input element `<input type='email'/>` except this use of it is replacing a StaticElement (a display) and not an input element?

Reply:
@zaliqarosli I'm confused how you phrased this question. The reason for the EmailElement is similar to why there was a request for a unique password component in the react login PR.

Reply:
is the password component you're mentioning an input element or a display element?

Reply:
Hey @zaliqarosli, the PR for upgrading the Login module to react is what i'm referring to and when I'm referencing the new password component in that PR (you suggested it). Instead of modifying one of the components to add a property that would change the text to be type "password" it was suggested to just make another component. I thought similar for creating an email component because of the type being email.

Reply:
okay great. I agree that we should be making separate components. Can you point me to where in the code in this PR that you are using this new EmailComponent that you added to jsx/Form.js?

Reply:
I'll delete the component then. As your above comment mentions it confuses the review process.

Reply:
no need to delete it. if it's not being used in this PR, then I think it's worth it for you to open up a separate PR for this new EmailElement component and include a use case there

Reply:
Hi @zaliqarosli, I removed the component from this PR.